oxidation/crates/platform-async/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "platform-async"
```

Comment:
`libparsec_platform_async` (the idea is it's easy to see the correspondance with the path `oxidation/libparsec/crates/platform_async` while still behind explicit about the fact this is not a 3rd party crate)

And we use underscore instead of dash (because it's what is used in the rust code when doing `use libparsec_platform_async::*`, so better stay consistent)

Reply:
it don't have `libparsec` in the path :p 

for the crate name, I have a lot of counter example that use the kebab case for it